supply-chain/audits.toml

```diff
@@ -233,6 +250,23 @@ criteria = "safe-to-deploy"
 version = "0.18.0"
 notes = "I am the author of this crate."
 
+[[audits.gimli]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+version = "0.26.1"
+notes = "This version is used in rust's libstd, so effectively we're already trusting it"
+
+[[audits.gimli]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.26.1 -> 0.27.0"
```

Comment:
Like the addr2line update above, it'd be good to have a brief `notes` here as well.

Reply:
Done too